include/seqan3/argument_parser/detail/format_help.hpp

```diff
@@ -680,6 +680,8 @@ class format_short_help : public format_help
      */
     void parse(argument_parser_meta_data const & parser_meta)
     {
+        meta = parser_meta;
+
```

Comment:
why do you set this member and not directly print the argument?

Reply:
Yeah @smehringer ... why are you doing this? :smiling_imp:

Reply:
Because I (@smehringer ) wanted to delegate to the `print_header` function for consistency. If we change something in the way we print the help page you would have to change it at two places.